webapp/channels/src/components/widgets/inputs/url_input/url_input.tsx

```diff
@@ -70,8 +70,8 @@ function UrlInput({
             onBlur(event);
         }
     };
-
-    const handleOnButtonClick = () => {
+    const handleOnButtonClick = (event: React.KeyboardEvent<HTMLButtonElement> | React.MouseEvent<HTMLButtonElement>) => {
+        event.stopPropagation();
```

Comment:
question: why the need of stop propagation here?

Reply:
Pressing Enter will close the modal for some reason, I think it's accepting the form by default.